rakelib/pod.rake

```diff
@@ -1,7 +1,9 @@
-namespace :pod do
-  desc 'Lint the Pod'
-  task :lint do |task|
-    Utils.print_info 'Linting the pod spec'
-    Utils.run(%Q(pod lib lint "#{POD_NAME}.podspec" --quick), task)
+if File.file?('Podfile')
```

Comment:
Given the nature of the task, we should rather test the presence of a `#{PODNAME}.podspec` rather than the presence of a `Podfile`.